test/test_uninstall.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+declare -a errors
+
+function error_and_proceed() {
+  errors+=("${1}")
+  echo -e "tfenv: ${0}: Test Failed: ${1}" >&2
+}
+
+function error_and_die() {
+  echo -e "tfenv: ${0}: ${1}" >&2
+  exit 1
+}
+
+[ -n "${TFENV_DEBUG}" ] && set -x
+source $(dirname $0)/helpers.sh \
+  || error_and_die "Failed to load test helpers: $(dirname $0)/helpers.sh"
+
+echo "### Uninstall local versions"
+cleanup || error_and_die "Cleanup failed?!"
+
+for v in 0.1.0; do
+  tfenv install ${v} || error_and_proceed "Install of version ${v} failed"
+done
+
+tfenv uninstall 0.1.0 || error_and_proceed "Uninstall of version ${v} failed"
+
+tfenv list | grep 0.1.0 && exit 1 || exit 0
```

Comment:
Please rewrite the test case `### Uninstall local versions` (L22~28) to the same approach of other tests such as the following.

```sh
v=0.1.0
(
  tfenv install ${v} || exit 1
  tfenv uninstall ${v} || exit 1
  tfenv list | grep ${v} || exit 1
) || error_and_proceed "Uninstall of version ${v} failed"
```